go.mod

```diff
@@ -106,6 +106,7 @@ require (
 	github.com/vivint/infectious v0.0.0-20190108171102-2455b059135b
 	github.com/yuin/gopher-lua v0.0.0-20180918061612-799fa34954fb // indirect
 	github.com/zeebo/admission v0.0.0-20180821192747-f24f2a94a40c
+	github.com/zeebo/assert v0.0.0-20181109011804-10f827ce2ed6
```

Comment:
do we really need this package?

why not github.com/stretchr/testify/assert?

Reply:
Nope, it picked it wrong